### ISSUE TITLE
Training loss explosion

### ISSUE THREAD

**Issue by hxiaozhi93** [mzolfaghari/ECO-efficient-video-understanding#20]
Dear sir, when I lately train the ECO network based on UCF101 dataset, the loss is always 87.3365 from the begining even I changed the learning rate and some other hyper parameters, also when I use the ECO_Lite_UCF101.caffemodel for tesing, it predicts all the objects to the same one class. I wonder whether you can give me any advice to solve the problem. Thank you.
